### PR TITLE
Update eth-trezor-keyring to @metamask/eth-trezor-keyring v1.0.0

### PR DESCRIPTION
**Pull request by danjm**
Updates the version to https://github.com/MetaMask/eth-trezor-keyring/releases/tag/v1.0.0

### CODE REVIEW

**Review by legobeat — DISMISSED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -1679,6 +1679,33 @@ __metadata:
...
+"@chainsafe/persistent-merkle-tree@npm:^0.4.2":
+  version: 0.4.2
+  resolution: "@chainsafe/persistent-merkle-tree@npm:0.4.2"
+  dependencies:
+    "@chainsafe/as-sha256": ^0.3.1
+  checksum: f9cfcb2132a243992709715dbd28186ab48c7c0c696f29d30857693cca5526bf753974a505ef68ffd5623bbdbcaa10f9083f4dd40bf99eb6408e451cc26a1a9e
+  languageName: node
+  linkType: hard
+
+"@chainsafe/ssz@npm:0.9.4":
```
[the remark this answers is not in the document]
Comment (danjm) [reply yarn.lock]: probably, yes

@brad-decker what was the plan for that?

[the remark this answers is not in the document]
Comment (brad-decker) [reply yarn.lock]: We are ignoring it in the bundler, (it wont' be included in bundles) -- also you can't merge this, the bump to ethereumjs/tx requires updates throughout the codebase that are expressed in #18302

**Comment by danjm** [comment]
blocked by https://github.com/MetaMask/metamask-extension/pull/18302

**Comment by brad-decker** [comment]
@danjm we can close this, #18302 includes trezor update

**Comment by danjm** [comment]
closing in favour of https://github.com/MetaMask/metamask-extension/pull/18302

**Pull request closed without merging**